test/Common.h

```diff
@@ -34,13 +34,13 @@ namespace solidity::test
 
 #ifdef _WIN32
 static constexpr auto evmoneFilename = "evmone.dll";
-static constexpr auto evmoneDownloadLink = "https://github.com/ethereum/evmone/releases/download/v0.11.0/evmone-0.11.0-windows-amd64.zip";
+static constexpr auto evmoneDownloadLink = "https://github.com/ethereum/evmone/releases/download/v0.12.0/evmone-0.12.0-windows-amd64.zip";
 #elif defined(__APPLE__)
 static constexpr auto evmoneFilename = "libevmone.dylib";
-static constexpr auto evmoneDownloadLink = "https://github.com/ethereum/evmone/releases/download/v0.11.0/evmone-0.11.0-darwin-x86_64.tar.gz";
+static constexpr auto evmoneDownloadLink = "https://github.com/ethereum/evmone/releases/download/v0.12.0/evmone-0.12.0-darwin-x86_64.tar.gz";
```

Comment:
The new link is broken. I guess the same problem as in https://github.com/ethereum/solidity/pull/15320#discussion_r1756760117?

Reply:
BTW, maybe we should simplify it and just link to the release page?

This platform guessing is not that robust and when it's wrong (e.g. for intel macs or some less common platform that's not win/mac/linux) it would probably be better to leave the choice up to the user than give the wrong link.

Reply:
Is the `evmoneDownloadLink` just for outputting the error message?